keyboards/anton/rules.mk

```diff
@@ -0,0 +1,30 @@
+# MCU name
+MCU = STM32F103
+
+# Bootloader selection
+BOARD = STM32F103C8_BOARD_FOR_ANTON
```

Comment:
Is this actually needed?  Most of the settings are set by default.

Reply:
I tried the `STM32_F103_STM32DUINO` board and I got a build pass. So I changed it to `BOARD = STM32_F103_STM32DUINO` in the latest commit.